packages/mc-scripts/config/create-webpack-config-for-development.js

```diff
@@ -55,8 +55,8 @@ module.exports = ({
   entry: {
     app: [
       // Ship a few polyfills by default
-      require.resolve('./polyfills'),
-      require.resolve('babel-polyfill'),
+      require.resolve('core-js/stable'),
+      require.resolve('regenerator-runtime/runtime'),
```

Comment:
More on this [here](https://github.com/zloirock/core-js/blob/master/docs/2019-03-19-core-js-3-babel-and-a-look-into-the-future.md#babelpolyfill) and [here](https://babeljs.io/blog/2019/03/19/7.4.0).

Reply:
Thanks for the explanation.
> Don't forget install those dependencies directly!

Have we already done that?

Reply:
Oh course. Why would I forget that. _cough_ fc5448f. Thanks.

Reply:
Sorry, I meant `core-js` and  `regenerator-runtime`
as suggested in https://github.com/zloirock/core-js/blob/master/docs/2019-03-19-core-js-3-babel-and-a-look-into-the-future.md#babelpolyfill
> Don't forget install those dependencies directly!
`npm i --save core-js regenerator-runtime`

Reply:
Yes, they're both added.

Reply:
both 
- regenerator-runtime
- core-js

was in the dependencies tree, but not directly dependencies for `@commercetools-frontend/mc-scripts`, but since we now require them directly we should add them to `mc-scripts`

Reply:
Ah. My bad. This should solve this, right? 0b17450